design/overview.md

```diff
@@ -41,7 +42,8 @@ Key concepts that we mention below will be defined here:
 * `EthBlockDelay` - Is a agreed upon number of Ethereum blocks all oracle attestations are delayed by. No `Orchestrator` will attest to have seen an event occur on Ethereum until this number of blocks has elapsed as denoted by their trusted Ethereum full node. This should prevent short forks form causing disagreements on the Cosmos side. The current value being consdiered is 50 blocks.
 * `Last ValSet` - The `Last Valset` is the last validator set known to have been accepted by the Ethereum chain. When a new ValSet update is submitted it is first `Confirmed` making it possible to submit and then `Observed` updating the value of `Last ValSet` in the store.
 * `Confirmed` - Many actions in Peggy require RLP encoded signatures to be submitted by the `Orchestrators`, an operation is `Confirmed` when it is possible to execute it on Ethereum by collecting and submitting these RLP encoded signatures representing 66% of all voting power encoded in the `Last Valset`.
-* `Observed` - events on Ethereum are considered `Observed` when 66% of the active Cosmos validator set during a given block has submitted an oracle message attesting to seeing the event. Note this is distinct from `Confirmed`! At any given time the set of who can `Observe` an event and who can `Confirm` and event will be slightly different. Since `Confirmed` is based off of the `Last ValSet` and `Observed` is always up to date with the latest validator set.
+* `Observed` - events on Ethereum are considered `Observed` when 66% of the active Cosmos validator set during a given block has submitted an oracle message attesting to seeing the event. Note this is distinct from `Confirmed`! At any given time the set of who can `Observe` an event and who can `Confirm` and event will be slightly different. Since `Confirmed` is based off of the `Last ValSet` and `Observed` is always up to date with the latest validator set. `Confirmed` has to do with bringing information from Cosmos to Ethereum, while `Observed` has to do with bringing information from Ethereum to Cosmos.
```

Comment:
this is good :+1:

Reply:
I don't want to block this PR but WDTY if we combine "66% of the active Cosmos validator" And ">50 % of the active validator count" ? this would give power to the smaller validators?